### ISSUE TITLE
Fine-tuning wav2veq seq2seq model ( hyperparameters)

### ISSUE THREAD

**Issue by sanakhamekhem** [pytorch/fairseq#3886]
Hello,

I have pretrained a wav2vec model using my own data (a set of 11000 samples, each sample have a frame number going from 1200 up to 2000 frames). It is not audo data. I have obtained a good accuracy of 99%.

Now, I'm would like to perform the fine tuning (1000 samples) using th following command line:

python train.py --distributed-world-size 1 --max-epoch 200 --batch-size 64 --distributed-port -1 "${out_manifest_path_tuning}" --save-dir $dirfinetuning --fp16 \
	--autoregressive --valid-subset valid --no-epoch-checkpoints --no-epoch-checkpoints --eval-wer --num-workers 1 \
	--max-update 10000 --sentence-avg --task audio_finetuning --arch wav2vec_seq2seq --w2v-path $dirPretrained/checkpoint_best.pt \
	--labels ltr --apply-mask --mask-selection static --mask-other 0 --mask-length 10 --mask-prob 0.65 --layerdrop 0.1 \
	--mask-channel-selection static --mask-channel-other 0 --mask-channel-length 64 --mask-channel-prob 0.004 \
	--feature-grad-mult 0.0 --freeze-finetune-updates 0 --validate-after-updates 100 --optimizer adam \
	--adam-betas '(0.9, 0.98)' --adam-eps 1e-08 --lr  0.0005 --lr-scheduler tri_stage --warmup-steps 2000 --hold-steps 8000 \
	--decay-steps 10000 --final-lr-scale 0.05 --final-dropout 0.0 --dropout 0.0 --activation-dropout 0.1 --criterion cross_entropy \
	--attention-dropout 0.0 --seed 2337 --log-format json --min-sample-size 1200 --log-interval 500 --ddp-backend no_c10d  --tensorboard-logdir $TENSORBOARD_LOGDIR_TUNING

 

However, The **wer on valid set is equal to 100%** during the tuning. I think that I should adjust some **hyperparameters**. I will appreciate any help in this field.

Thank you in advance.
